### PR TITLE
Ideas for extended learning

### PR DESCRIPTION
**Pull request by craigjbass**

### CODE REVIEW

File: ideas/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Go back over workshops
+
+Can you apply the knowledge you picked up in a previous workshop or seminar in a new project.
+
+Do you need to follow the guide, or can you remember what to do?
+
+## Shadow others
+
+If possible, arrange to shadow a team actively developing software.
```
Comment (danielburnley) [inline ideas/README.md]: This is a cool idea, is it worth letting them know which teams are open to this ahead of time? Be easier for them if let them know they're not gonna be intruding

File: ideas/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+## Code Review
+
+Go and review some code. (You don't even need to comment).
```
Comment (danielburnley) [inline ideas/README.md]: "Come join team akzo's gitlab"

* cult intensifies *

File: ideas/README.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+* Read the code + understand it.
+* Do you think it's well designed.
+* Clone the project, try to run it locally.
+
+## Work on a project
+
+If you have a burning desire to build something, do so.
```
Comment (danielburnley) [inline ideas/README.md]: How would I get feedback on the thing that I'm building? Is there a slack room I can ask for help/post my PRs?

**Comment by danielburnley** [comment]
All good ideas :+1: Only comments are making sure it's clear to academy peeps that they won't be intruding/there's a place they can seek some of these things out

File: ideas/README.md (hunk 1 above)
Comment (craigjbass) [reply ideas/README.md]: I've asked leads to invite the academy out for lunch, so a good time to ask I guess

File: ideas/README.md (hunk 2 above)
Comment (craigjbass) [reply ideas/README.md]: In the back of my mind I'm conscious that this content isn't just for Academy

Comment (danielburnley) [reply ideas/README.md]: Oh yeah these comments aren't necessarily for actual noting down, just things to maybe mention in slack and offline when introducing this

**Pull request merged**